src/gui/search/searchwidget.cpp

```diff
@@ -381,3 +382,9 @@ void SearchWidget::closeTab(int index)
 
     delete tab;
 }
+
+void SearchWidget::closeAllTabs()
+{
+    while(m_allTabs.size())
+        closeTab(0);
```

Comment:
You should loop to close the right-most tab due to `m_allTabs` is a list and it will start behave like vector in Qt6.

Reply:
Okay, thank you